style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+  width: 100%;
```

Comment:
Be careful about using width: 100% in your * selector, this will cause things, such as inline elements, to behave as block elements.